models/usermodel.js

```diff
@@ -3,7 +3,11 @@ var mongoose    = require("mongoose"),
 
 var UserSchema  = new mongoose.Schema({
     username    : String,
-    password    : String
```

Comment:
Add age, gender, description (which would be a small paragraph about him), birthdate, role (which would be admin/user).